### ISSUE TITLE
[cli] Different password when using -C

### ISSUE THREAD

**Issue by Rudloff** [lesspass/lesspass#146]
With lesspass-cli 4.0.1, I get different results when using `-C`:
```
lesspass foo bar
master password: baz
=>~pof5X@YaVu4Qa
```

```
lesspass -C foo bar
master password: baz
p:8C_o'zA@4Gd1HX
```

**Comment by guillaumevincent** [comment]
@Rudloff order matters

I use meow for parsing options and there is no `booleanish` option.

So `lesspass -C foo bar` set `C` option to `foo`.
For now you need to put `-C` at the end.

```
lesspass foo bar -C
master password: baz
Copied to clipboard
=>~pof5X@YaVu4Qa^C
```

I will try to fix lesspass-cli with another cli helper or create a PR to fix https://github.com/sindresorhus/meow/issues/44

**Comment by Rudloff** [comment]
Oh right, sorry. I'm really used to having flags before arguments (like cp, chmod, etc.).

**Comment by guillaumevincent** [comment]
@Rudloff don't be sorry, it's a bug and argument should work before and after

**Comment by guillaumevincent** [comment]
@Rudloff fixed in version 4.0.2

`npm update -g lesspass-cli`
